From 2360c08176eba7b7d8708921be4cd86c43ad32b5 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Tue, 28 Feb 2017 17:55:11 -0600
Subject: [PATCH] fix symlinks creation

Signed-off-by: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
---
 lib/Makefile | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/lib/Makefile b/lib/Makefile
index 02a72cf..7491d99 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -121,9 +121,8 @@ endif
 
 # Create symlinks to DSO
 ifeq ($(SHARED),y)
-	ln -f -s $(LIBDIR)/$(LIB).so.0 $(LIBDIR)/$(LIB).so
-	ln -f -s $(LIBDIR)/$(LIBNAME) $(LIBDIR)/$(LIB).so.0
-	ldconfig
+	ln -f -s $(LIB).so.0 $(LIBDIR)/$(LIB).so
+	ln -f -s $(LIBNAME) $(LIBDIR)/$(LIB).so.0
 endif
 
 uninstall:
-- 
2.12.0

