--- intel-cmt-cat-1.0.0/Makefile~	2017-05-18 14:14:15.000000000 +0000
+++ intel-cmt-cat-1.0.0/Makefile	2017-05-26 13:47:37.142323127 +0000
@@ -36,7 +36,7 @@
 ###############################################################################
 
 # XXX: modify as desired
-PREFIX ?= /usr/local
+PREFIX ?= /usr/
 export PREFIX
 
 ifdef DEBUG
--- intel-cmt-cat-1.1.0/lib/Makefile~	2017-07-20 13:38:36.000000000 +0000
+++ intel-cmt-cat-1.1.0/lib/Makefile	2017-07-25 15:53:08.009920562 +0000
@@ -114,14 +114,6 @@
 	install -m 0644 $(HDR) -D $(HDR_DIR)/$(HDR)
 endif
 
-# Create symlinks to DSO
-ifeq ($(SHARED),y)
-	cd $(LIB_INSTALL_DIR); \
-		ln -f -s $(LIB).so.$(VERSION) $(LIB).so.$(SO_VERSION); \
-		ln -f -s $(LIB).so.$(SO_VERSION) $(LIB).so
-	ldconfig
-endif
-
 uninstall:
 	-rm $(LIB_INSTALL_DIR)/libpqos*
 ifeq ($(SHARED),y)
